tests/unit_tests/testsettingsholder.cpp

```diff
@@ -117,4 +117,22 @@
 #include "settingslist.h"
 #undef SETTING
 
+#define EXPERIMENTAL_FEATURE(experimentId, experimentDescription, \
```

Comment:
Why is EXPERIMENTAL_FEATURE being re-defined for these tests (instead of using feature.h/cpp)?

Reply:
To test the experimental feature APIs inside the SettingsHolder.